NEWS.md

```diff
@@ -1,3 +1,16 @@
+## 30.0.1 2020-05-28
+
+ * [RMB-627](https://issues.folio.org/browse/RMB-627) Fix stringType false seems to produce invalid SQL
```

Comment:
```suggestion
 * [RMB-627](https://issues.folio.org/browse/RMB-627) Generate valid SQL for index with "stringType": false
```

Reply:
ok. Making a new branch with that change.